schedule/yast/modify_existing_partition_sle.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name:           modify_existing_partition
+description:    >
+  Installation where we modify some pre-existing partitions. Must depend on some create_hdd test suite.
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning/modify_existing_partition
+  - installation/installer_timezone
+  - installation/hostname_inst
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  # Called on ARCH: s390x
+  - {{reconnect_mgmt_console}}
+  # Called on BACKEND: qemu
+  - {{grub_test}}
+  - installation/first_boot
+  - console/validate_modify_existing_partition
+
+test_data:
+    disk: 'vda'
+    existing_partition: 'vda2'
+    mount_point: '/'
+    fs_type: 'ext4'
+    part_size: '10GiB'
```

Comment:
```suggestion
    partition_size: '10'
    partition_unit:
        binary: GiB
        human_readable: G
```

Reply:
a little more fancy and clear.

Reply:
AFAIS lsblk uses GiB but prints G. We cannot tell lsblk to use a specific unit, like MiB, it's either human readable "ibit" unit or byte, so if the user writes 10 000MB in test data, we'll end up comparing 10000 with 9.5: specifying the unit in the schedule does not help, unless we create a method to convert any unit to byte. But maybe I didn't get your idea ?

Reply:
you're right so then my first code suggestion should work as both value and unit should be together as you mentioned.